bootstrap.sh

```diff
@@ -1 +1,36 @@
 #!/bin/bash
+
+# Create namespaces
+echo "Creating namespaces..."
+kubectl create namespace mysql || echo "Namespace 'mysql' already exists."
+kubectl create namespace todoapp || echo "Namespace 'todoapp' already exists."
+
+# Deploy MySQL resources
+echo "Deploying MySQL resources..."
+kubectl apply -f ./.infrastructure/namespace.yml
+kubectl apply -f ./.infrastructure/st-configMap.yml
+kubectl apply -f ./.infrastructure/st-secret.yml
+kubectl apply -f ./.infrastructure/st-service.yml
+kubectl apply -f ./.infrastructure/statefulSet.yml
```

Comment:
Ensure that the `statefulSet.yml` file is correctly configured and exists in the specified path. This file is crucial for deploying the MySQL StatefulSet.